mini_project_2/parser.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+import re
+import xml.etree.ElementTree as ElementTree
+
+
+def parse(xml_string):
+    term_string = parse_terms(xml_string)
+    create_terms_file(term_string)
+
+    pdate_string = parse_pdates(xml_string)
+    create_pdates_file(pdate_string)
+
+    price_string = parse_prices(xml_string)
+    create_prices_file(price_string)
+
+    ad_string = parse_ads(xml_string)
+    create_ads_file(ad_string)
+
+
+def parse_terms(xml_string):
+    terms = list()
+
+    elem = ElementTree.fromstring(xml_string)
+    root = ElementTree.ElementTree(elem).getroot()
+
+    for ad in root.findall("ad"):
+        aid = ad.find("aid").text
+
+        title = ad.find("ti").text
+        title = re.sub("['\"]", "", title)
+
+        for token in re.split(r"(?!-)\W", title):
```

Comment:
my not functionize this?

easy to read, debug and whatnot

Reply:
I'm not sure what you're asking. Feel free to push whatever changes necessary before merging.